debian/rules

```diff
@@ -11,6 +11,8 @@ extraopts += -DWITH_CEPHFS_JAVA=ON
 extraopts += -DCMAKE_INSTALL_LIBDIR=/usr/lib
 extraopts += -DCMAKE_INSTALL_LIBEXECDIR=/usr/lib
 extraopts += -DCMAKE_INSTALL_SYSCONFDIR=/etc
+NPROC=$(shell nproc)
```

Comment:
see https://www.debian.org/doc/debian-policy/ch-source.html, we might want to put something like
```sh
ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
  NUMJOBS = $(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
  extraopts += -DBOOST_J=$(NUMJOBS)
endif
```

and let the pbuilder decide the parallel level of the build, currently we use "-j`grep -c processor /proc/cpuinfo`", see https://github.com/ceph/ceph-build/blob/master/ceph-build/build/build_deb#L83.